### PR TITLE
chore: add required flag for Route name field

### PR DESCRIPTION
**Pull request by okaybase**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**
1. add required flag for Route name field
2. fix hosts field for FAQ(update host to hosts)

image
image

- [x] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

fix/resolve

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by juzhiyuan — COMMENTED** [review]
Changes LGTM, and please add the related test case for your changes.

File: web/src/pages/Route/components/DebugViews/AuthenticationView.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ const AuthenticationView: React.FC<RouteModule.DebugViewProps> = (props) => {
               onChange={(event) => {
                 const currentValue = event.target.value;
                 setAuthType(currentValue);
-                props.form.setFieldsValue({ authType: currentValue });
+                props.form.setFieldsValue({ autyType: currentValue });
```
Comment (juzhiyuan) [inline web/src/pages/Route/components/DebugViews/AuthenticationView.tsx]: May I know why change this back?

Comment (okaybase) [reply web/src/pages/Route/components/DebugViews/AuthenticationView.tsx]: the whole feature has been remove in current version

**Review by juzhiyuan — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Comment by liuxiran** [comment]
Please add the related test case for your changes, thanks very much~

Hope this will help you to add your test case : https://stackoverflow.com/questions/55516990/cypress-testing-pseudo-css-class-before

**Comment by okaybase** [comment]
> Please add the related test case for your changes, thanks very much~
> 
> Hope this will help you to add your test case : https://stackoverflow.com/questions/55516990/cypress-testing-pseudo-css-class-before

@liuxiran Only document changes are involved this time.

**Comment by juzhiyuan** [comment]
> @liuxiran Only document changes are involved this time.

And also the `required` mark.

**Comment by liuxiran** [comment]
> > @liuxiran Only document changes are involved this time.
> 
> And also the `required` mark.

Yep, need test case for this `required` mark, even though it is a small improvement, we also want it to be a valid feature. This will also make your pr merged without any worries, thanks

**Comment by okaybase** [comment]
> > > @liuxiran Only document changes are involved this time.
> > 
> > 
> > And also the `required` mark.
> 
> Yep, need test case for this `required` mark, even though it is a small improvement, we also want it to be a valid feature. This will also make your pr merged without any worries, thanks

the test case has been added. @juzhiyuan @liuxiran

**Review by liuxiran — APPROVED** [review]

**Comment by liuxiran** [comment]
> the test case has been added. @juzhiyuan @liuxiran

merged, thanks for your contribution~!!

**Pull request merged**